setup.py

```diff
@@ -114,11 +114,11 @@ def get_git_sha() -> str:
         "redis",
         "selenium>=3.141.0",
         "simplejson>=3.15.0",
-        "slackclient==2.5.0",  # PINNED! slack changes file upload api in the future versions
+        "slack_sdk>=3.1.1",
```

Comment:
Can we add a `,4` here, just in case?
```suggestion
        "slack_sdk>=3.1.1, <4",
```